sdk/src/main/java/com/mapbox/maps/viewannotation/ViewAnnotationManager.kt

```diff
@@ -155,6 +157,26 @@ interface ViewAnnotationManager {
    */
   fun getViewAnnotationUpdateMode(): ViewAnnotationUpdateMode
 
+  /**
+   * Get the [CameraOptions] object bound to specified annotations on the MapView.
+   *
+   * Using this method with GlobeProjection might lead to incorrect results.
+   *
+   * @param annotations list of views needs to fit in the visible frame.
+   * @param edgeInsets paddings to apply.
+   * @param bearing camera bearing to apply.
+   * @param pitch camera pitch to apply.
+   *
+   * @return [CameraOptions] object, Will be null if empty annotations provided.
+   *
+   */
+  fun cameraForAnnotations(
```

Comment:
State behaviour in documentation about framing invisible / gone views? How will it behave in general, did you test it?